sorc/ncep_post.fd/INITPOST.F

```diff
@@ -1,46 +1,24 @@
 !> @file
-!
-!> SUBPROGRAM:    INITPOST    INITIALIZE POST FOR RUN
-!!   PRGRMMR: RUSS TREADON    ORG: W/NP2      DATE: 93-11-10
-!!     
-!! ABSTRACT:  THIS ROUTINE INITIALIZES CONSTANTS AND
-!!   VARIABLES AT THE START OF AN ETA MODEL OR POST 
-!!   PROCESSOR RUN.
-!!
-!!   THIS ROUTINE ASSUMES THAT INTEGERS AND REALS ARE THE SAME SIZE
-!!     
-!! PROGRAM HISTORY LOG:
-!!   93-11-10  RUSS TREADON - ADDED DOCBLOC
-!!   98-05-29  BLACK - CONVERSION OF POST CODE FROM 1-D TO 2-D
-!!   99-01 20  TUCCILLO - MPI VERSION
-!!   01-10-25  H CHUANG - MODIFIED TO PROCESS HYBRID MODEL OUTPUT
-!!   02-06-19  MIKE BALDWIN - WRF VERSION
-!!   02-08-15  H CHUANG - UNIT CORRECTION AND GENERALIZE PROJECTION OPTIONS
-!!   21-03-11  Bo Cui - change local arrays to dimension (im,jsta:jend)
-!!     
-!! USAGE:    CALL INIT
-!!   INPUT ARGUMENT LIST:
-!!     NONE     
-!!
-!!   OUTPUT ARGUMENT LIST: 
-!!     NONE
-!!     
-!!   OUTPUT FILES:
-!!     NONE
-!!     
-!!   SUBPROGRAMS CALLED:
-!!     UTILITIES:
-!!       NONE
-!!     LIBRARY:
-!!       COMMON   - CTLBLK
-!!                  LOOKUP
-!!                  SOILDEPTH
-!!
-!!    
-!!   ATTRIBUTES:
-!!     LANGUAGE: FORTRAN
-!!     MACHINE : CRAY C-90
-!!
+!> @brief initpost() initializes post for run.
```

Comment:
There needs to be two doxygen blocks, one for the file, one for the subprogram.

Reply:
Thank you so much for reviewing, Ed! I updated them all. Please have a look at them.